torch/csrc/onnx/diagnostics/diagnostics.h

```diff
@@ -0,0 +1,71 @@
+#pragma once
+#include <torch/csrc/onnx/diagnostics/generated/rules.h>
+#include <torch/csrc/utils/pybind.h>
+#include <string>
+
+namespace torch {
+namespace onnx {
+namespace diagnostics {
+
+/**
+ * @brief Level of a diagnostic.
+ * @details The levels are defined by the SARIF specification, and are not
+ * modifiable. For alternative categories, please use Tag instead.
+ * @todo Introduce Tag to C++ api.
+ */
+enum class Level : uint32_t {
```

Comment:
Just for my learning, no changes needed: do we need uint32 or can a smaller type do it?

Reply:
Good point, I think `uint8_t` will do.